app/controllers/users_controller.rb

```diff
@@ -3,6 +3,7 @@
 require 'uri'
 
 class UsersController < ApplicationController
+  before_action :configure_permitted_parameters, if: :devise_controller?
```

Comment:
Are you sure this filter is used, as I think `devise_controller?` is going to return `false` for `UsersController`?